internal/kafka/internal/services/kafka.go

```diff
@@ -1015,6 +1015,12 @@ func buildManagedKafkaCR(kafkaRequest *dbapi.KafkaRequest, kafkaConfig *config.K
 	if err != nil {
 		return nil, errors.NewWithCause(errors.ErrorGeneral, err, "unable to list kafka request")
 	}
+	labels := map[string]string{
+		"bf2.org/kafkaInstanceProfileQuotaConsumed": strconv.Itoa(k.QuotaConsumed),
+	}
+	if strings.Contains(api.AllInstanceTypeSupport.String(), kafkaRequest.InstanceType) {
```

Comment:
This is not totally accurate because `AllInstanceTypeSupport` is only used if the data plane config file does not explicitly set the values for supported_instance_types.
If we assume we will never run the new code with "eval" in any manually configured data plane cluster then I guess this might be enough, as long as we delete this code later.
An alternative to just compare with "eval" (or a constant that contains it).

Reply:
> An alternative to just compare with "eval" (or a constant that contains it).

+1, I think it is better if the comparison is done against a know instance type i.e `eval`.

Reply:
+1 to compare against `eval`, however you will have to add the constant back (types.EVAL)